installer/bundle/bundle_skel.sh

```diff
@@ -805,6 +829,23 @@ if [ "$installMode" = "I" -o "$installMode" = "U" ]; then
         echo "sed was not installed, installation cannot continue. Please install sed."
         cleanup_and_exit $INSTALL_SED
     fi
+    curl_installed
+    if [ $? -ne 0 ]; then
+        if [ -z "${forceFlag}" ]; then
+            echo "Error: curl is not installed, installation cannot continue."
```

Comment:
It is actually not needed at all, echo command would not throw an error. 
It was probably copy paste for the python, but we never redirect the output from echo command